content/_changelogs/12.0.0.md

```diff
@@ -1,20 +1,191 @@
 ## 12.0.0
 
-_Released MM/DD/YYYY_
+_Released 12/06/2022_
+
+**Summary:**
```

Comment:
In the 11.0 release, I just copied the splash screen verbiage to the summary here. I figured the splash screen really is a summary of the release.